wiki/Contests/KCC/2/en.md

```diff
@@ -40,6 +40,8 @@ The **Korean Charting Contest 2022** (***KCC2022***) is an osu!mania mapping con
 ## Links
 
 - [Forum thread](https://osu.ppy.sh/community/forums/topics/1507792)
+- [Results sheet](https://docs.google.com/spreadsheets/d/1IBIBR7fjgqM90sQEiBWg_J1x1IZqD1PoCevrm7RHZ1g/edit?usp=sharing)
+- [Entries](https://lulu.s-ul.eu/PkrKU3sx)
```

Comment:
just saying, some day this link *may* eventually expire for one reason or another